source/index.ts

```diff
@@ -54,6 +54,14 @@ class Stdin extends EventEmitter {
 	pause() {
 		// Do nothing
 	}
+
+	ref() {
```

Comment:
Nice addition! I also had to change something else in this class for it to work. In our project I copy and pasted it. Here it is.

```ts
export class Stdin extends EventEmitter {
  isTTY: boolean
  data: string | null = null

  constructor(options: {isTTY?: boolean} = {}) {
    super()
    this.isTTY = options.isTTY ?? true
  }

  write = (data: string) => {
    this.data = data
    this.emit('readable')
  }

  setEncoding() {}
  setRawMode() {}
  ref() {}
  unref() {}
  read: () => string | null = () => {
    const data = this.data

    this.data = null

    return data
  }
}
```

Basically we no longer listen to the `data` event so we need new logic to use the `readable` event.

Reply:
Awesome, thank you! I updated the PR with your changes